code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -262,10 +262,10 @@
 		if(D)
 			D.Evolve(level_min, level_max)
 
-
 /datum/chemical_reaction/mix_virus/mix_virus_2
 	name = "Mix Virus 2"
 	id = "mixvirus2"
+	results = list(/datum/reagent/blood = null)
```

Comment:
Just set this to results = null instead, test it, make sure it wont runtime

Reply:
Cool, yeah it runs fine